### ISSUE TITLE
Stale on C8 plot

### ISSUE THREAD

**Issue by jess4140** [madMAx43v3r/chia-gigahorse#69]
I've been having issues with stales for the last few days. As soon as I pass 200TB of plots, C8 stales occur. I have regular plots along with the new C8 ones.

My setup is:

Ryzen 9 3900
64GB RAM
Nvidia Quadro K6000
55 drives connected through PCIe expanders and SATA
Most of the stales are from the 19 drives that were re-plotted with C8 compression.
any sugestion ?

**Comment by madMAx43v3r** [comment]
Wait, K6000 is not supported, so your CPU is probably overloaded

**Comment by madMAx43v3r** [comment]
Also check your partial diff

**Comment by spleen911** [comment]
Quadro P600 (2GB) with ~20TB (259 plots @ C7) is giving me about 1% stales.

DELL R620 2xE5-2697v2, 512GB 1333MT, Windows 2022 Server + flexfarmer 2.3.2 inside WSL2 (Ubuntu). GPU memory usage is flat at 1.7GB. I haven't see utilization go over 71%, but it never stales when watching it. Disk IO is very light with the hardware being 24xSAS-6G drives in a RAID5. Before you flame me for RAID5, it sustained 1GB/s (big B) writes when moving the plots over 10Gbps network. Reads are not even 25% that level of load.

??? Wondering if the P600 is the bottleneck and if a T400 would get me over the hump ??? 20TB is not a lot of space so a bit surprised since a K2200 should be able to farm ~200TB if I'm reading the chart correctly.

I have a T600 on order for my other DELL R620 that farms ~200TB uncompressed. I plan to re-plot all of that once the T600 is installed.

**Comment by madMAx43v3r** [comment]
Yeah it's strange, keep monitoring it and see if it is consistent.

**Comment by spleen911** [comment]
I noticed the GPU was installed in a 16x physical, but 8x bandwidth slot so I moved it. Letting it run 24 hours to see if stales decrease. After that, I will disable hardware acceleration and see what the CPUs can do by themselves. Maybe I got a bo-bo GPU. For comparison, the other R620 farming uncompressed is <0.1% stale with 10x the space utilized.

<img width="549" alt="GPU_Slot" src="https://user-images.githubusercontent.com/68015090/231134252-615a3059-b348-4d33-9aff-feb6b146ae89.png">

<img width="542" alt="pizza" src="https://user-images.githubusercontent.com/68015090/231134934-4e30f940-447c-4f0b-9202-32cfa1b83a51.png">

**Comment by madMAx43v3r** [comment]
Wait, what's your partial diff setting?

**Comment by spleen911** [comment]
I increased difficulty to 4 and in the last 7 hours ... 0 stales from any farmer == happy.

The 5th farmer is online although I stopped plotting so the temp wouldn't run out of space while I sleep.
734 plots @ C8 yielding `space=56.18 TB space-effective=79.93 TB`

**Comment by madMAx43v3r** [comment]
> Honestly, I didn't know that was something I should update/size properly. Do you have a suggestion?

Yes, it's very important. Recommended is 18, but for small farms it will make the statistics bad.

**Comment by xusanduo08** [comment]
so , where and how to set parital difficulty? I got warning below:
![image](https://user-images.githubusercontent.com/17930163/234890169-e3a051a1-c1ff-447f-a4f4-9d81917ea656.png)
